src/app/games/lines-of-action/tests/LinesOfActionRules.spec.ts

```diff
@@ -272,7 +251,8 @@ describe('LinesOfActionRules', () => {
         ];
         const state: LinesOfActionState = new LinesOfActionState(board, 0);
         expect(LinesOfActionRules.getVictory(state)).toEqual(MGPOptional.of(Player.ONE));
-        expect(minimax.getBoardValue(new MGPNode(null, null, state)).value).toBe(Player.ONE.getVictoryValue());
+        expect(minimax.getBoardValue(new LinesOfActionNode(state)).value)
```

Comment:
constantify node (or if you even feel like it, RulesUtils.expectToBeVictoryFor :p )

Reply:
done